app/api/services/brief_overview.py

```diff
@@ -63,26 +25,50 @@ def brief_contains_any_optional_fields(self, brief, optional_fields):
         return False
 
     def is_brief_section_complete(self, brief, section):
+        from ...brief_utils import determine_required_fields
         if section['required']:
-            return self.brief_contains_all_required_fields(brief, section['required'])
+            required_fields = determine_required_fields(brief, section, enforce_required=False)
+            return self.brief_contains_all_required_fields(brief, required_fields)
         else:
             return self.brief_contains_any_optional_fields(brief, section['optional'])
 
     def get_publish_links(self, brief):
         links = []
         draft_brief = brief.status == 'draft'
-
+        brief_builder_paths = {
+            'How long your brief will be open': '{path}/edit/how-long-your-brief-will-be-open/requirementsLength',
+            'Description of work': '{path}/description-of-work',
+            'Location': '{path}/edit/location/location',
+            'Question and answer session details': '{path}/question-and-answer-session-details',
+            'Role': '{path}/edit/role/title',
+            'Shortlist and evaluation process': '{path}/shortlist-and-evaluation-process',
+            'Title': '{path}/edit/title/title',
+            'Who can respond': '{path}/edit/who-can-respond/specifySeller',
+            'Description of training': '{path}/description-of-training',
+            'Timeframes, location and budget': '{path}/timeframes,-location-and-budget',
+            'Evaluate responses': '{path}/evaluate-responses',
+            'Additional information': '{path}/additional-information',
+            'Organisation': '{path}/edit/organisation/organisation',
+            'Approach to market': '{path}/approach-to-market',
+            'Summary': '{path}/edit/summary/summary'
+        }
         # Get the sections key from the JSON schema generated by the frameworks repo
         schema_sections = get_sections('briefs-{}-{}'.format(brief.framework.slug, brief.lot.slug))
-
         for section in schema_sections:
-            path = self.get_path_for_brief_link(brief, section['name']) if draft_brief else None
-            links.append(self.build_brief_link(self.is_brief_section_complete(brief, section), path, section['name']))
+            if section.get('editable', True):
```

Comment:
hang about - that will always be `True` whether there's `editable` or not?

Reply:
editable is a new property I added for display only sections.  currently all sections are editable